docs/source/CONTRIBUTING.rst

```diff
@@ -161,7 +161,7 @@ releases, and to discuss the technical and operational direction of the project
 and sub-projects.
 
 Please see the
-`wiki <https://wiki.hyperledger.org/display/fabric/Maintainer+Meetings>`__
+`wiki <https://wiki.hyperledger.org/display/fabric/Contributor+Meetings>`__
```

Comment:
Thanks Pam, could you make the same update in the paragraph above and section heading?